lib/kino/smart_cell/chart_builder.ex

```diff
@@ -102,7 +111,15 @@ defmodule Kino.SmartCell.ChartBuilder do
 
   def handle_event("update_field", %{"field" => field, "value" => value}, ctx) do
     parsed_value = parse_value(field, value)
-    ctx = update(ctx, :fields, &Map.put(&1, field, parsed_value))
+
+    ctx =
+      if field in @shared_fields do
+        update(ctx, :fields, &Map.put(&1, field, parsed_value))
+      else
+        updated_layers = put_in(ctx.assigns.fields["layers"], [Access.at(0), field], parsed_value)
+        update(ctx, :fields, &Map.put(&1, "layers", updated_layers))
+      end
+
     broadcast_event(ctx, "update", %{"fields" => %{field => parsed_value}})
```

Comment:
I think this now sends layer field updates that look like top-level field updates on the client.